server/integration_tests/test_data/e2e_answer_places/whatistheemissionsinthesecounties/chart_config.json

```diff
@@ -15,39 +15,21 @@
                       "sort": "DESCENDING"
                     },
                     "comparisonPlaces": [
-                      "geoId/06105",
```

Comment:
this got moved down because dc/topic/Emissions is a better match for the query & no data existed for that topic before